find_primes.py

```diff
@@ -1 +1,29 @@
+#num1 is the original number, num2 is the prime number, num3 is all of the primes,
+num1 = int(input("Enter any positive integer: "))
+num2 = num1 + 1
+num3 = []
+count = 0
+
+def find_prime(a):
```

Comment:
The name find_prime implies that the function finds and returns a prime number. However, this function actually checks whether a given number is prime or not. I suggest changing the name of the function to be is_prime instead and also changing the name of the parameter from a to candidate, because that is the candidate that is being checked.